src/Pages/Shared/_Layout.cshtml

```diff
@@ -26,7 +26,6 @@
 </head>
 <body>
     <div class="container">
-        <partial name="_CookieConsentPartial" optional />
```

Comment:
also we can remove this because we're not using it and it's giving us an error in the IDE